eng/scripts/tests/Update-SdkVersion.tests.ps1

```diff
@@ -0,0 +1,97 @@
+Set-StrictMode -Version 3
+
+AfterAll {
+    ."$PSScriptRoot/Update-SdkVersion.constants.ps1"
+    CheckoutChangedFiles
+}
+
+Describe 'Update-SdkVersion ExistingBeta' {
+    BeforeEach {
+        ."$PSScriptRoot/Update-SdkVersion.constants.ps1"
+        Set-Content -Path $AzVersionLocation -Value $AzVersionContentBeta
+    }
+
+    It "Increments beta version when no parameters are applied" {
```

Comment:
Wow, what type of syntax is this and what does `It` mean in powershell? This reads like self-documenting English prose :)

Reply:
This is [pester](https://pester.dev/docs/usage/test-file-structure), a test framework for PowerShell. Since functions in PowerShell are called with spaces and named or positional parameters this calls the function `It` with the first parameter of that string and the second parameter as the code block.